source/server/http_time_tracking_filter.cc

```diff
@@ -0,0 +1,68 @@
+#include "server/http_time_tracking_filter.h"
```

Comment:
Yes I think that would make sense. Alternatively, we could keep this directory just for extensions and their configuration, and re-home everything else that's in there so far: configuration.h/cc (helpers), well_known_headers.h (http header name definitions). Should I file an issue to track this?

Reply:
An issue to track this would be great.  I'm happy to defer to you on what makes the most sense.  The `filters` subdirectory was just what made sense to me on a quick lookthrough.